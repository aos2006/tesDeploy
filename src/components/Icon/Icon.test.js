import Icon from './Icon';
import { snapShotRender } from '../../utils/snapShotUtil';

describe('<Icon />', () => {
  const mock = {
    width: 30,
    height: 40,
    value: 'add',
  };

  snapShotRender('Icon without Props', Icon, mock, true);
});
