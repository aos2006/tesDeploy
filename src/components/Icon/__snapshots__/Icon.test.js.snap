// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Icon /> Корректно отображается без props - Icon without Props 1`] = `
<svg
  className="icon icon-"
  height={10}
  width={undefined}
>
  <use
    xlinkHref="#icon-"
  />
</svg>
`;

exports[`<Icon /> Корректно отображается с props - Icon without Props 1`] = `
<svg
  className="icon icon-add"
  height={40}
  width={30}
>
  <use
    xlinkHref="#icon-add"
  />
</svg>
`;
